content/articles/intro-to-emberjs/index.md

```diff
@@ -0,0 +1,155 @@
+---
+layout: engineering-education
+status: publish
+published: false
+url: /intro-to-emberjs
+title: Introduction to Ember JS
+description: Ember.js is a free JavaScript web framework that follows the Model–View–View Model (MVVM) architecture. By combining popular idioms and best practises into the framework, developers may design scalable single-page web apps.
+author: dev-verma
+date: 2022-03-08 T00:00:00-13:23
+topics: [languages]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/intro-to-emberjs
+    alt: Ember JS
+---
+
+
+###  Ember JS
+### Introduction
+Ember.js is a free JavaScript client-side framework for designing web apps that is open source. It is a component-service framework that focuses on the overall online application development experience, eliminating minor variations between apps while remaining a current and lightweight overlay on top of native JavaScript. What does a component-service framework mean? Components, like those provided by other front-end frameworks like React, Vue, and Angular, are separate bundles of action, style, and HTML.
+### Use cases
+EmberJS is a good choice for apps that want to have one or both of the following characteristics:
+
+1. Single-page apps, such as native-like web apps and progressive web apps (PWAs)
+    1. Ember performs best when it is used to build your complete application's front end..
+
+2. Improving technology stack cohesion across many teams
+    1. "Best practices" supported by the community enable speedier long-term development.
+
+    2. Ember features well-defined norms that are beneficial for enforcing uniformity and assisting team members in swiftly catching up.
+### Installing the Ember tooling
+Ember builds and scaffolds sections of your application using a command-line interface (CLI) tool.
+
+1.  Before you can install `ember-cli`, you'll need to have node and npm installed. If you don't already have node and npm installed, go here to learn how to do so.
```

Comment:
`npm installed, go here to learn how to do so.`

Guess you have missed out the link